src/core_plugins/metrics/public/lib/__test__/create_number_handler.js

```diff
@@ -0,0 +1,29 @@
+import sinon from 'sinon';
```

Comment:
i think this file belongs to `components/vis_editor/lib` as well as some other files in this folder.

i would try to keep the files that are imported only by some part as close to the import as possible, to make it easier to understand the structure of the code. in this folder i would only put what is actually shared across everything on this level.